deployments/r/config/common.yaml

```diff
@@ -71,3 +71,9 @@ jupyterhub:
       guarantee: 512M
       limit: 1G
     image: {}
+
+    custom:
+      group_profiles:
+        1527100: # ESPM 157, Fall 2023, issue #4822
```

Comment:
i think this needs to be `course::1537100`.  @ryanlovett to confirm his docs are correct...  :)

https://docs.datahub.berkeley.edu/en/latest/admins/howto/course-config.html#defining-group-profiles

Reply:
Yep! Using the course ID alone was a previous method.